edx-platform/pearson-ironwood/lms/templates/static_templates/server-error.html

```diff
@@ -0,0 +1,41 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%inherit file="../main.html" />
+
+<%def name="get_tech_support_link_address()"><%
+    return configuration_helpers.get_value('TECH_SUPPORT_LINK', settings.TECH_SUPPORT_LINK)
+%></%def>
+
+<main id="main" aria-label="Content" tabindex="-1">
+    <section class="outside-app">
+        <h1>
+            <%block name="pageheader">
+                % if page_header:
+                    ${page_header}
+                % else:
+                    ${Text(_(u"There has been a 500 error on the {platform_name} servers")).format(
+                    platform_name=HTML("<em>{platform_name}</em>").format(platform_name=Text(static.get_platform_name()))
+                    )}
+                % endif
+            </%block>
+        </h1>
+        <p>
+            <%block name="pagecontent">
+                % if page_content:
+                    ${page_content}
+                % else:
+                    ${Text(_('Please wait a few seconds and then reload the page. If the problem persists, please visit {start}Support{end}.')).format(
+                        start=HTML('<a href="{support_link}">').format(support_link=Text(get_tech_support_link_address())),
+                        end=HTML('</a>')
+                        )
+                    }
```

Comment:
```suggestion
                    )}
```
Last suggestion... @luismorenolopera 
Applies for all three files.